test/go/iterative_verifier_integration_test.go

```diff
@@ -1,293 +0,0 @@
-package test
-
-import (
-	"database/sql"
-	"testing"
-
-	"github.com/Shopify/ghostferry"
-	"github.com/Shopify/ghostferry/testhelpers"
-	"github.com/siddontang/go-mysql/schema"
-	"github.com/stretchr/testify/assert"
-)
-
-func TestHashesSql(t *testing.T) {
-	columns := []schema.TableColumn{schema.TableColumn{Name: "id"}, schema.TableColumn{Name: "data"}, schema.TableColumn{Name: "float_col", Type: schema.TYPE_FLOAT}}
-	pks := []uint64{1, 5, 42}
-
-	sql, args, err := ghostferry.GetMd5HashesSql("gftest", "test_table", "id", columns, pks)
-
-	assert.Nil(t, err)
-	assert.Equal(t, "SELECT `id`, MD5(CONCAT(MD5(COALESCE(`id`, 'NULL')),MD5(COALESCE(`data`, 'NULL')),MD5(COALESCE((if (`float_col` = '-0', 0, `float_col`)), 'NULL')))) "+
-		"AS row_fingerprint FROM `gftest`.`test_table` WHERE `id` IN (?,?,?) ORDER BY `id`", sql)
-	for idx, arg := range args {
-		assert.Equal(t, pks[idx], arg.(uint64))
-	}
-}
-
-func TestVerificationFailsDeletedRow(t *testing.T) {
```

Comment:
This test tests if a row within the reverifyStore will be detected during cutover if it was deleted on the target.

The InlineVerifier test that effectively covers this case is when the test corrupts a BinlogStreamer insert and the InlineVerifier detect it during the cutover:
https://github.com/Shopify/ghostferry/blob/b8147c5ff8653099f6c5e24128aa6fd41ff401e1/test/integration/inline_verifier_test.rb#L115-L138